packages/main/src/DatePicker.js

```diff
@@ -199,6 +200,26 @@ const metadata = {
 			type: Object,
 		},
 	},
+
+	slots: /** @lends  sap.ui.webcomponents.main.DatePicker.prototype */ {
+		/**
+		 * Defines the value state message that will be displayed as pop up under the <code>ui5-datepicker</code>.
+		 * <br><br>
+		 *
+		 * <b>Note:</b> If not specified, a default text (in the respective language) will be displayed.
+		 * <br>
+		 * <b>Note:</b> The <code>valueStateMessage</code> would be displayed,
+		 * when the <code>ui5-input</code> is in <code>Information</code>, <code>Warning</code> or <code>Error</code> value state.
```

Comment:
ui5-input

Reply:
- Docs is fixed
- Fixed the js error
- In phone mode it shouldn't be in the dialog.